### PR TITLE
Correct AOT instructions MacOS and Win

### LINKED ISSUE #29112: Add MacOS instructions to AOT
See https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 8839fa1e-7c10-4b22-d03d-669906a2b9ea
* Version Independent ID: 8839fa1e-7c10-4b22-d03d-669906a2b9ea
* Content: [ASP.NET Core support for native AOT](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/native-aot?branch=pr-en-us-29062&view=aspnetcore-8.0&tabs=netcore-cli)
* Content Source: [aspnetcore/fundamentals/native-aot.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/native-aot.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @mitchdenny
* Microsoft Alias: **midenn**

---
[Associated WorkItem - 117425](https://dev.azure.com/msft-skilling/Content/_workitems/edit/117425)

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #29112 

The instructions work the same using CLI for MacOS and Windows, so no MacOS tab is needed.  I clarified a bit on the cli vs VS template description.

Tested with VS 2022 Preview/.NET 8 and project generation on MacOS with CLI.  Thanks Andy for testing on MacOS for compile/publish.  

Verified diff in the launchSettings.json were correct.  They are on both Win and MacOS and VS and CLI.

Note: VS for Mac 2022 preview is not working with .NET 8 preview yet so there is no VS for Mac specific instruction.  

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/native-aot.md](https://github.com/dotnet/AspNetCore.Docs/blob/8a97c5ae5a73e09a07ae70ae8ff508230b9460b4/aspnetcore/fundamentals/native-aot.md) | [ASP.NET Core support for native AOT](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/native-aot?branch=pr-en-us-29574) |

### CODE REVIEW

File: aspnetcore/fundamentals/aot/native-aot-tutorial.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Create an ASP.NET Core API app that is configured to work with native AOT:
 Run the following commands:
 
 ```dotnetcli
-dotnet new api --aot -o MyFirstAotWebApi && cd MyFirstAotWebApi
+dotnet new api -aot -o MyFirstAotWebApi && cd MyFirstAotWebApi
```
Comment (tdykstra) [inline aspnetcore/fundamentals/aot/native-aot-tutorial.md:52]: I just tried this command and it works prefixed with two hyphens and it doesn't work prefixed by one hyphen. Can you try again to see if -aot works and if it does, what SDK version are you using?

Comment (wadepickett) [reply aspnetcore/fundamentals/aot/native-aot-tutorial.md:52]: OK, thanks Tom, maybe a version difference?.  I tested it on both win and MacOS using CLI with one hyphen and it worked and using two hyphens did not.  I'll test again.  I was using 8.0.100-preview.3.23178.7 and VS 2022 17.7.0 Preview 2.0.  What do you have?

Comment (wadepickett) [reply aspnetcore/fundamentals/aot/native-aot-tutorial.md:52]: Output for CLI below: 1st with two hyphens and 2nd with one hyphen which runs successfully:

```
dotnet new api --aot -o MyFirstAotWebApi
Error: Invalid option(s):
--aot
   '--aot' is not a valid option

For more information, run:
   dotnet new api -h

For details on the exit code, refer to https://aka.ms/templating-exit-codes#127

dotnet new api -aot -o MyFirstAotWebApi
The template "ASP.NET Core API" was created successfully.
```

Comment (wadepickett) [reply aspnetcore/fundamentals/aot/native-aot-tutorial.md:52]: I should have been at ASP.NET Core Runtime 8.0.0-preview.5.23302.2 which went up on the 13th.  I'll go back and test.

Comment (wadepickett) [reply aspnetcore/fundamentals/aot/native-aot-tutorial.md]: @tdykstra,  the command did indeed change with the last version.  I changed it back to one hyphen, thanks!  I forgot you added a commit and made the change back locally and committed.  Sorry I bypassed yours, not intentional.  Should be right now.

**Review by mitchdenny — APPROVED** [review]
Lgtm

**Pull request merged**